scripts/integration/default_set.yaml

```
@@ -121,6 +121,12 @@ tests:
         input: data/json/conn.log.json.gz
       - command: export -N ascii 'duration > 6s'
       - command: export -N ascii '#time >= 2011-08-15T03:48'
+  Node suricata alert:
+    tags: [node, import-export, suricata, eve]
+    steps:
+      - command: import -N -b suricata
```

```suggestion
      - command: import -N suricata
```

`-N` implies `-b`, no?

---

In a way it does. I removed all `-b`s from commands with `-N`.